cli/src/commands/util/mod.rs

```diff
@@ -45,6 +46,9 @@ pub(crate) enum UtilCommand {
     ConfigSchema(UtilConfigSchemaArgs),
     Exec(UtilExecArgs),
     Gc(UtilGcArgs),
+    /// No longer supported; use `jj util install-man-pages` instead. This stub
```

Comment:
I feel like this command should be hidden in the Clap sense. This should also remove it from the docs. TODO: Find the right syntax.

Reply:
I think addign `#[command(hide = true)]` here will do it.

Reply:
Yes, good point, thanks for raising.